Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -1270,6 +1270,20 @@ void KeyframeEffect::apply(RenderStyle& targetStyle, const Style::ResolutionCont
     setAnimatedPropertiesInStyle(targetStyle, *computedTiming.progress, *computedTiming.currentIteration);
 }
 
+bool KeyframeEffect::isRunningAccelerated() const
+{
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
```

Comment:
is it time to remove this compile guard?

Reply:
Not just yet.